examples/ci/bazel.yml

```diff
@@ -0,0 +1,25 @@
+options:
+  env:
+    GOPATH: /workspace
+steps:
+- id: get-workspace-gen
+  uses: docker://tikhoplav/go
+  args: [go, get, github.com/jin/create-bazel-workspace]
+
+- id: install-gen
+  uses: docker://tikhoplav/go 
+  args: [go, install, github.com/jin/create-bazel-workspace]
+
+- id: gen-workspace
+  uses: docker://tikhoplav/go
+  args: [./bin/create-bazel-workspace]
```

Comment:
these steps could be consolidated in a single one:

```yaml
- id: setup-workspace
  uses: docker://tikhoplav/go
  runs: [bash, -uxec]
  env:
    GOPATH: /workspace
  args:
  - |
    go get ...
    go install ..
    bin/create-bazel-workspace
```

or alternatively, a docker image for `create-bazel-workspace` could be created and uploaded to https://github.com/getpopper/library, and that image can be invoked instead of the base go image.

One question: is `create-bazel-workspace` idempotent? what happens if it is executed multiple times?

Reply:
another alternative would be to assume that the workspace already exists and only provide as example the bazel commands. This is what the other CI examples do (for example the NodeJS one assumes a packages.json file already exists).

Reply:
yes, I would better assume that the BUILD and WORKSPACE files exist to preserve step idempotency